receiver/k8sobjectsreceiver/README.md

```diff
@@ -50,6 +50,7 @@ the K8s API server. This can be one of `none` (for no auth), `serviceAccount`
 - `label_selector`: select objects by label(s)
 - `field_selector`: select objects by field(s)
 - `interval`: the interval at which object is pulled, default 60 minutes. Only useful for `pull` mode.
+- `exclude_deleted`: whether the object's deleted updates should be dropped. Only usable in `watch` mode.
```

Comment:
Why do we need to introduce a config for deleted events only? Shouldn't we have a generic configuration to limit the set of watched events?

Reply:
You're right, we could make this a generic list of accepted types, which would default to ADDED, UPDATED, and DELETED.

Reply:
Does k8s allow for custom types returned for a watch? I wouldn't want to miss any that aren't the three I mentioned.

Reply:
> Does k8s allow for custom types returned for a watch? I wouldn't want to miss any that aren't the three I mentioned.

I'm not sure. @jinja2, do you know by chance?

Reply:
Maybe the best would be to have it be an exclude list and if it is empty then we exclude nothing. Then we don't have to worry about guessing what is allowed.

Reply:
https://github.com/kubernetes/apimachinery/blob/v0.28.1/pkg/watch/watch.go#L40 should be the possible types

Reply:
Ok so we can actually use the predefined `EventType`

Reply:
Updated to allow excluding any watch type.